src/py-opentimelineio/opentimelineio/adapters/fcp_xml.py

```diff
@@ -561,9 +561,7 @@ class FCP7XMLParser:
         1. Inheritance
         2. The id Attribute
 
-    .. seealso:: https://developer.apple.com/library/archive/documentation/\
-            AppleApplications/Reference/FinalCutPro_XML/Basics/Basics.html\
-            #//apple_ref/doc/uid/TP30001154-TPXREF102
+    .. seealso:: https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/FinalCutPro_XML/Basics/Basics.html#//apple_ref/doc/uid/TP30001154-TPXREF102 # noqa
```

Comment:
is seealso a typo or a keyword?

Reply:
Unfortunately it's a keyboard with a typo in it, see https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso.